conda/integrated/build.sh

```diff
@@ -47,32 +47,11 @@ else
   export NO_CUDA=1
 fi
 
-
-###########################################################
-# Build Caffe2
-###########################################################
-cmake_args=()
-cmake_args+=("-DCMAKE_INSTALL_PREFIX=$PREFIX")
-
-# Build Caffe2
-mkdir -p caffe2_build && pushd caffe2_build
-cmake "${cmake_args[@]}" $CAFFE2_CMAKE_ARGS ..
```

Comment:
Heads up that switching from calling cmake manually to calling FULL_CAFFE2=1 setup.py will get rid of the ability to forward user cmake flags from scripts/build_anaconda.sh (and so some of the flags passed in our CI script will be ignored and that might be a little confusing debugging). A good fix for this IMO is to change tools/build_pytorch_libs.sh to also allow propogating cmake flags, but we can do that after this is merged.